test/test_pipeline/components/training/base.py

```diff
@@ -21,10 +21,11 @@ class BaseTraining(unittest.TestCase):
 
     def prepare_trainer(self,
                         trainer: BaseTrainerComponent,
-                        task_type: int):
+                        task_type: int,
+                        epochs=50):
         if task_type in CLASSIFICATION_TASKS:
             X, y = make_classification(
-                n_samples=5000,
+                n_samples=500,
```

Comment:
Can we define a constant variable for this so that we can easily change the value by one line modification in the future?